KunligiOnline/kunligiconnect
# Adds redux-devtools and Navbar test for loading

## Description:
- Adds redux-devtools and Navbar test for loading

## Changes I Made:
- Added redux-devtools-extension package and wrapper for state debugging.
- Created Navbar test.

## How to Test:
- npm run test should now run tests. Redux DevTools should now show state in browser.
